koap/js_tests/decodeTcp.test.js

```diff
@@ -0,0 +1,39 @@
+let koapModule = require('../build/distributions/koap')
+const koap = koapModule.com.juul.koap
+
+let kotlinModule = require('../build/publications/npm/js/node_modules/kotlin')
+const kotlin = kotlinModule
+
+test('TCP empty payload :: correctly decoded', () => {
```

Comment:
The testing format I'm going with atm is:

Function being tested is notated by the file name. In this case, decodeTcp
'TCP empty payload' is denoting the special input conditions of the test
'correctly decoded' is denoting what results are being tested for.